### ISSUE TITLE
Floating point exception (core dumped)

### ISSUE THREAD

**Issue by kasteric** [QwenLM/Qwen-Image#75]
Floating point exception (core dumped) 这个报错是什么原因啊
Fetching 36 files: 100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 36/36 [04:28<00:00,  7.47s/it]
Loading checkpoint shards: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 9/9 [00:03<00:00,  2.32it/s]
Loading checkpoint shards: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 4/4 [00:01<00:00,  2.47it/s]
Loading pipeline components...: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 6/6 [00:06<00:00,  1.12s/it]
pipeline loaded



  0%|                                                                                                                                                                                                                | 0/50 [00:00<?, ?it/s]


Floating point exception (core dumped)

**Comment by ZetangForward** [comment]
Please make sure to align with the environment we listed in the README. 

If you are still encountering this issue despite meeting the specified requirements, plz provide your GPU model, along with detailed environment information, including package versions, CUDA, and PyTorch versions, etc, so we can further help you locate the problem.

**Comment by kaixinbear** [comment]
Same Issue!
cuda version: 
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2024 NVIDIA Corporation
Built on Thu_Mar_28_02:18:24_PDT_2024
Cuda compilation tools, release 12.4, V12.4.131
Build cuda_12.4.r12.4/compiler.34097967_0


pip list:
Package                       Version
----------------------------- ------------
absl-py                       2.1.0
accelerate                    1.4.0
addict                        2.4.0
aiofiles                      23.2.1
aiohappyeyeballs              2.4.6
aiohttp                       3.11.13
aiosignal                     1.3.2
aliyun-python-sdk-core        2.16.0
aliyun-python-sdk-kms         2.16.5
annotated-types               0.7.0
antlr4-python3-runtime        4.9.3
anyio                         4.8.0
apptools                      5.3.1
asttokens                     3.0.0
attrdict                      2.0.1
attrs                         25.1.0
av                            14.2.0
binpacking                    1.5.2
blinker                       1.9.0
ccimport                      0.4.4
certifi                       2025.1.31
cffi                          1.17.1
charset-normalizer            3.4.1
click                         8.1.8
colorama                      0.4.6
coloredlogs                   15.0.1
colorlog                      6.9.0
comm                          0.2.2
ConfigArgParse                1.7.1
configobj                     5.0.9
contourpy                     1.3.1
cpm-kernels                   1.0.11
crcmod                        1.7
cryptography                  44.0.2
cumm-cu124                    0.7.11
cycler                        0.12.1
dacite                        1.9.2
dash                          3.0.4
datasets                      3.2.0
decorator                     5.2.1
diffusers                     0.36.0.dev0
dill                          0.3.8
distro                        1.9.0
docstring_parser              0.16
einops                        0.8.1
embreex                       2.17.7.post6
envisage                      7.0.4
executing                     2.2.0
fastapi                       0.115.11
fastjsonschema                2.21.1
ffmpy                         0.5.0
filelock                      3.14.0
fire                          0.7.0
flash_attn                    2.7.4.post1
Flask                         3.0.3
flatbuffers                   25.2.10
fonttools                     4.56.0
freetype-py                   2.5.1
frozenlist                    1.5.0
fsspec                        2024.9.0
future                        1.0.0
gradio                        5.17.1
gradio_client                 1.7.1
grpcio                        1.70.0
h11                           0.14.0
hf-xet                        1.1.8
httpcore                      1.0.7
httpx                         0.28.1
huggingface-hub               0.34.3
humanfriendly                 10.0
hydra-core                    1.3.2
idna                          3.10
imageio                       2.37.0
imagesize                     1.4.1
importlib_metadata            8.6.1
iopath                        0.1.10
ipython                       9.3.0
ipython_pygments_lexers       1.1.1
ipywidgets                    8.1.7
itsdangerous                  2.2.0
jedi                          0.19.2
jieba                         0.42.1
Jinja2                        3.1.5
jiter                         0.8.2
jmespath                      0.10.0
joblib                        1.4.2
jsonschema                    4.24.0
jsonschema-specifications     2025.4.1
jupyter_core                  5.8.1
jupyterlab_widgets            3.0.15
kiwisolver                    1.4.8
lark                          1.2.2
lazy_loader                   0.4
lxml                          5.4.0
manifold3d                    3.1.1
mapbox_earcut                 1.0.3
Markdown                      3.7
markdown-it-py                3.0.0
MarkupSafe                    2.1.5
matplotlib                    3.6.0
matplotlib-inline             0.1.7
mayavi                        4.8.3
mdurl                         0.1.2
mmcv                          2.1.0
mmdet                         3.3.0
mmengine                      0.10.4
model-index                   0.1.11
modelscope                    1.23.1
mpmath                        1.3.0
msgspec                       0.19.0
multidict                     6.1.0
multiprocess                  0.70.16
narwhals                      1.44.0
nbformat                      5.10.4
nest-asyncio                  1.6.0
networkx                      3.4.2
ninja                         1.11.1.3
nltk                          3.9.1
nodeenv                       1.9.1
numpy                         1.26.4
nvidia-cublas-cu12            12.1.3.1
nvidia-cuda-cupti-cu12        12.1.105
nvidia-cuda-nvrtc-cu12        12.1.105
nvidia-cuda-runtime-cu12      12.1.105
nvidia-cudnn-cu12             9.1.0.70
nvidia-cufft-cu12             11.0.2.54
nvidia-curand-cu12            10.3.2.106
nvidia-cusolver-cu12          11.4.5.107
nvidia-cusparse-cu12          12.1.0.106
nvidia-cusparselt-cu12        0.6.2
nvidia-nccl-cu12              2.20.5
nvidia-nvjitlink-cu12         12.4.127
nvidia-nvtx-cu12              12.1.105
omegaconf                     2.3.0
onnxruntime                   1.22.0
open3d                        0.19.0
openai                        1.65.2
opencv-python                 4.10.0.84
opendatalab                   0.0.10
openmim                       0.3.9
openxlab                      0.1.2
ordered-set                   4.1.0
orjson                        3.10.15
oss2                          2.17.0
packaging                     24.2
pandas                        2.2.3
parso                         0.8.4
pccm                          0.4.16
peft                          0.17.0
pexpect                       4.9.0
pillow                        11.1.0
pip                           25.0
platformdirs                  4.3.8
plotly                        6.1.2
plyfile                       1.1.2
portalocker                   3.1.1
prompt_toolkit                3.0.51
propcache                     0.3.0
protobuf                      5.29.3
psutil                        7.0.0
ptyprocess                    0.7.0
pure_eval                     0.2.3
puremagic                     1.29
pyarrow                       19.0.1
pybind11                      2.13.6
pyceres                       2.3
pycocotools                   2.0.10
pycollada                     0.9
pycolmap                      3.10.0
pycparser                     2.22
pycryptodome                  3.21.0
pydantic                      2.10.6
pydantic_core                 2.27.2
pydub                         0.25.1
pyface                        8.0.0
Pygments                      2.19.1
pyparsing                     3.2.1
pyquaternion                  0.9.9
python-dateutil               2.9.0.post0
python-multipart              0.0.20
pytz                          2023.4
PyVirtualDisplay              3.0
PyYAML                        6.0.2
qwen-vl-utils                 0.0.10
referencing                   0.36.2
regex                         2024.11.6
requests                      2.28.2
retrying                      1.4.0
rich                          13.4.2
rouge                         1.0.1
rpds-py                       0.25.1
rtree                         1.4.0
ruff                          0.9.9
safehttpx                     0.1.6
safetensors                   0.5.3
scikit-image                  0.25.2
scikit-learn                  1.7.0
scipy                         1.15.2
semantic-version              2.10.0
sentencepiece                 0.2.0
setuptools                    60.2.0
shapely                       2.1.1
shellingham                   1.5.4
shtab                         1.7.2
simplejson                    3.20.1
six                           1.17.0
sniffio                       1.3.1
sortedcontainers              2.4.0
spconv-cu124                  2.3.8
stack-data                    0.6.3
starlette                     0.46.0
svg.path                      6.3
sympy                         1.13.1
tabulate                      0.9.0
tensorboard                   2.19.0
tensorboard-data-server       0.7.2
termcolor                     3.1.0
terminaltables                3.1.10
threadpoolctl                 3.6.0
tifffile                      2025.6.11
tiktoken                      0.9.0
timm                          1.0.3
tokenizers                    0.21.0
tomlkit                       0.12.0
torch                         2.4.0
torchvision                   0.19.0
tqdm                          4.65.2
traitlets                     5.14.3
traits                        7.0.2
traitsui                      8.0.0
transformers                  4.49.0
transformers-stream-generator 0.0.4
trimesh                       4.6.12
triton                        3.0.0
trl                           0.15.2
typeguard                     4.4.3
typer                         0.15.2
typing_extensions             4.14.0
tyro                          0.9.24
tzdata                        2025.1
urllib3                       1.26.20
uvicorn                       0.34.0
vhacdx                        0.0.8.post2
viser                         0.2.23
vtk                           9.5.0
wcwidth                       0.2.13
websockets                    14.2
Werkzeug                      3.0.6
wheel                         0.45.1
widgetsnbextension            4.0.14
xvfbwrapper                   0.2.13
xxhash                        3.5.0
yapf                          0.43.0
yarl                          1.18.3
yourdfpy                      0.0.57
zipp                          3.21.0
zstandard                     0.23.0

**Comment by kasteric** [comment]
> Please make sure to align with the environment we listed in the README.
> 
> If you are still encountering this issue despite meeting the specified requirements, plz provide your GPU model, along with detailed environment information, including package versions, CUDA, and PyTorch versions, etc, so we can further help you locate the problem.

I solved this, it was because of the GPU issue, I have to download a specific cublas version

**Comment by kaixinbear** [comment]
Could you tell me which version of nvidia-cublas-cu12 do you use? I use the version 12.1.3.1

**Comment by kasteric** [comment]
> Could you tell me which version of nvidia-cublas-cu12 do you use? I use the version 12.1.3.1

Hi, if you are using the same version of GPU as I do, you can refer to https://blog.csdn.net/m0_46435741/article/details/146028843

**Comment by kaixinbear** [comment]
Thanks, I will try it.

**Comment by wendashi** [comment]
For H20, `pip install nvidia-cublas-cu12==12.5.3.2 -i https://pypi.tuna.tsinghua.edu.cn/simple requests` will be helpful.

**Comment by UnderElm** [comment]
> For H20, `pip install nvidia-cublas-cu12==12.5.3.2 -i https://pypi.tuna.tsinghua.edu.cn/simple requests` will be helpful.

you， my hero
